jcl/src/java.base/share/classes/openj9/internal/criu/InternalCRIUSupport.java

```diff
@@ -44,8 +46,8 @@ public final class InternalCRIUSupport {
 
 	private static void init() {
 		if (!initComplete) {
-			if (loadNativeLibrary()) {
-				criuSupportEnabled = isCRIUSupportEnabledImpl();
+			if (!loadNativeLibrary()) {
```

Comment:
We can leave library loading to CRIUSupport

Reply:
we can load the library lazily just before the checkpoint

Reply:
we will need to test the behvaiour on a system without libcriu installed

Reply:
It appears CRIU JVM won't start at a system w/o `libcriu.so`
```
failed to open </root/ive/criubuilds/jdk11-criu/lib/default/libjvm.so> - reason: <libcriu.so.2: cannot open shared object file: No such file or directory>
Failed to find VM - aborting
```

Reply:
does that happen with the changes I suggested above?

Reply:
```
ld /root/ive/criubuilds/jdk11-criu/lib/default/libjvm.so
ld: warning: libcriu.so.2, needed by /root/ive/criubuilds/jdk11-criu/lib/default/libjvm.so, not found (try using -rpath or -rpath-link)
```
I think lazy loading `j9criu29` won't help this though it is right thing to do (PR to be updated).

Reply:
what if you did an excplicit library load with `System.loadlibrary(.)` would it throw an exception? or hard fail with an `ld` failure?

Reply:
otherwise we may need to search the library load path to determine if libcriu exists

Reply:
Delayed `j9criu` loading until `checkpointJVMImpl()`, it doesn't help at a machine w/o `libcriu` installed.
https://openj9-artifactory.osuosl.org/artifactory/ci-openj9/Build_JDK11_x86-64_linux_criu_Personal/24/OpenJ9-JDK11-x86-64_linux_criu-20220418-175638.tar.gz
```
./jdk/bin/java -version
failed to open </root/ive/criubuilds/0418/jdk/lib/default/libjvm.so> - reason: <libcriu.so.2: cannot open shared object file: No such file or directory>
Failed to find VM - aborting
```
Might be related to
https://github.com/eclipse-openj9/openj9/blob/e14e2105f9709630f92b3028ec8b5d91d427db9d/runtime/cmake/platform/toolcfg/gnu.cmake#L44

I think this error is not related to https://github.com/eclipse-openj9/openj9/issues/14789, can be investigated further in a separated issue instead.